ExampleApp/Views/ProductDetailViewController.swift

```diff
@@ -55,11 +55,22 @@ class ProductDetailViewController: BaseViewController {
                   let text = textField.text,
                   let url = URL(string: text) else { return }
 
+//            let toolbarUI = ThreeDSToolbarCustomization(
+//                backgroundColorHex: "#FFFFFF",
+//                headerText: "Secure Checkout",
+//                buttonText: "Close",
+//                textFontName: "Arial-BoldMT",
+//                textColorHex: "#000000",
+//                textFontSize: 20
+//            )
+//
+//            let threeDSUICustomization = ThreeDSUICustomization(toolbarCustomization: toolbarUI)
```

Comment:
Curious, why are we commenting this block?

Reply:
I wasn't sure if i should apply customization or just leave it commented as an example. I'll uncomment it.